lib/optimizely/bucketer.rb

```diff
@@ -92,9 +92,7 @@ def bucket(experiment, bucketing_id, user_id)
       end
 
       # Handle the case when the traffic range is empty due to sticky bucketing
-      if variation_id == ''
-        @config.logger.log(Logger::DEBUG, 'Bucketed into an empty traffic range. Returning nil.')
-      end
+      @config.logger.log(Logger::DEBUG, 'Bucketed into an empty traffic range. Returning nil.') if variation_id == ''
```

Comment:
Same comment here, is there a way to invert this relationship? Seems like it makes it harder to read